ui/src/main/AndroidManifest.xml

```diff
@@ -102,8 +102,7 @@
         </receiver>
 
         <receiver
-            android:name=".model.TunnelManager$IntentReceiver"
-            android:permission="${applicationId}.permission.CONTROL_TUNNELS">
```

Comment:
Why did we remove this?

Reply:
As it was internal permission and was unnecessary for code to function properly. So I have removed it, and it was functioning as before.